src/mysqlserver.py

```diff
@@ -7,9 +7,9 @@
 import string
 
 from mysql.connector import connect, Error
-from typing import Union
 
 logger = logging.getLogger(__name__)
+MYSQL_VERSION = "8.0.26"
```

Comment:
will need to remember to keep this in sync with the uploaded resource (image)

Reply:
Sure!